File: C#/date_time.cs
using System;

namespace TimeAndDate {
    public class App {
        static void Main(string[] args) {
            var time_date = DateTime.Now;
            string date = time_date.ToString("MM/dd/yyyy");
            string time = time_date.ToString("hh:mm:ss");
            Console.WriteLine("Date: " + date + "\nTime: " + time);
        }
    }
}

File: Java/DateTime.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class DateTime {
    public static void main(String[] args) {
        LocalDateTime local_time = LocalDateTime.now();
        String date = local_time.format(DateTimeFormatter.ofPattern("MM/dd/yyyy"));
        String time = local_time.format(DateTimeFormatter.ofPattern("HH:mm:ss"));
        System.out.print("Date: " + date + "\nTime: " + time);
    }
}